docs/index.rst

```diff
@@ -18,13 +18,43 @@ If you need any of those services for your machine, ``pro`` is the right tool fo
 
 ``pro`` is already installed on every Ubuntu system. Try it out by running ``pro help``!
 
+-----
+
+.. grid:: 1 1 2 2
+
+   .. grid-item-card:: **Tutorials**
```

Comment:
Adding this gutter value evens out the spacing between the cards horizontally and vertically and I think looks a tad better
```suggestion
.. grid:: 1 1 2 2
   :gutter: 3

   .. grid-item-card:: **Tutorials**
```